### ISSUE TITLE
test_updown fails intermittently on Python 2.6

### ISSUE THREAD

**Issue by roolebo** [svinota/pyroute2#271]
Steps to reproduce:
`while true;do tox -e py26 -- /bin/sh -c "'cd tests && nosetests general/test_ipdb.py:TestExplicit.test_updown'"; done`

1 out of 20 times it fails:

```
E
======================================================================
ERROR: test_ipdb.TestExplicit.test_updown
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/pyroute2/.tox/py26/lib/python2.6/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/root/pyroute2/tests/general/test_ipdb.py", line 663, in test_updown
    self.ip.interfaces[self.ifd].commit()
  File "/root/pyroute2/.tox/py26/lib/python2.6/site-packages/pyroute2/ipdb/interface.py", line 940, in commit
    raise error
CommitException: ipaddr setup error

----------------------------------------------------------------------
Ran 1 test in 0.249s

FAILED (errors=1)
```

**Comment by svinota** [comment]
Is NetworkManager running on the system?

**Comment by roolebo** [comment]
Yep, it's running on the system. How does it impact the test execution?

**Comment by svinota** [comment]
Then I'm surprised, why not more tests are failing.

From time to time the NetworkManager adds back removed IP addresses, unpredictably brings test dummy interfaces up/down, interferes with the testing in other ways.

I discussed that with a NM developer, but I have no patience to build the NM master to get all the logs etc, so I just gave up.

Maybe it would be a good idea to detect the NM interference by PID in RTNL messages and report to the logs.
